lib/pharos/configuration/host.rb

```diff
@@ -20,7 +20,7 @@ class Host < Dry::Struct
       attr_accessor :os_release, :cpu_arch, :hostname, :api_endpoint, :private_interface_address, :checks
 
       def to_s
-        address
+        hostname || address
```

Comment:
This can be quite long in aws (fqdn). Should we just use a short hostname always?

Reply:
Done